R/api.R

```diff
@@ -0,0 +1,105 @@
+library(optiSel)
+library(nprcgenekeepr)
+library(DBI)
+library(jsonlite)
+# Returns a connection instance to a PG server
+connect_to_pg <- function() {
+  pg_host <- Sys.getenv("POSTGRES_HOST", "localhost")
+  con <- dbConnect(RPostgres::Postgres(),dbname = 'herdbook',
+                   host = get("pg_host"),
+                   port = 5432,
+                   user = 'herdbook',
+                   password = 'insecure')
+  return (con)
+}
```

Comment:
You should probably make R read the same settings file that python does, otherwise this will break once the database is secured.

Reply:
Agreed, and the best way of doing is so probably to put these variables in a environment file and add  
```
        env_file:
            - .envfile
```

to all the services.

Reply:
Okey that sounds like a separate PR. Can you make a issue about that? But I agree